src/Symfony/Component/Notifier/Bridge/Mercure/composer.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "symfony/mercure-notifier",
+    "type": "symfony-bridge",
+    "description": "Symfony Mercure Notifier Bridge",
+    "keywords": ["mercure", "notifier"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Mathias Arlaud",
+            "email": "mathias.arlaud@gmail.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=7.2.5",
+        "ext-json": "*",
+        "psr/container": "^1.0",
+        "symfony/mercure-bundle": "^0.2",
```

Comment:
A component must not depend on a bundle because otherwise we cannot use it in apps that don't use Symfony as a full-stack framework. Can we change this to `symfony/mercure` instead?

Reply:
Actually, that was my first approach. But @dunglas suggested me to use the bundle instead to benefit from DI and configuration. Maybe we can rely on `symfony/mercure` only and use `symfony/mercure-bundle` interfaces only if that bundle is detected?
WDYT?

Reply:
Unless I'm mistaken, there is no usage of MercureBundle in the bridge actually (only "soft" usages in FrameworkBundle, which is fine) so removing that dependency should work without changing anything.

Requiring symfony/mercure would allow documenting what the publisher locator is about though (i.e. `@param ContainerInterface $publisherLocator A container that holds PublisherInterface instances`).

Reply:
Yes indeed!
And we can suggest installing `symfony/mercure-bundle` during the bridge composer installation?

Reply:
We don't use composer `suggest` entries (people don't read those usually). 
Would it make sense to throw an exception like [this one](https://github.com/symfony/symfony/blob/5.x/src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php#L217) in FrameworkExtension instead? In case the bridge is available but not mercure-bundle